pkg/services/app/add.go

```diff
@@ -77,6 +77,32 @@ type AddParams struct {
 	GitProviderToken string
 }
 
+const (
+	DefaultName           = ""
+	DefaultURL            = ""
+	DefaultPath           = "./"
+	DefaultBranch         = "main"
+	DefaultDeploymentType = "kustomize"
+	DefaultChart          = ""
+	DefaultPrivateKey     = ""
+	DefaultAppConfigURL   = ""
+	DefaultDryRun         = false
+	DefaultAutoMerge      = false
+)
+
+func (a AddParams) PopulateDefaultValues() {
```

Comment:
We should be using this `func` somewhere right?

Reply:
Correct, this meant to be used when we implement Add in the api-service.

Reply:
Can we use it now in the `.Add()` service? The API will probably just consume that service.

Reply:
Can we use it now in the `.Add()` service? The API will probably just consume that service.

Reply:
Also, this doesn't need  a receiver. We want to pass in a pointer to the `AddParams` because we are modifying it.

Reply:
@josecordaz PTAL at these comments ^^

Reply:
@jpellizzari if definitely can use it in the `.Add()` service but that will overwrite any parameter set by the user using different values from the default ones.  Maybe I need to populate the fields only if they are empty ?

Reply:
I guess I am confused because this is a function that never gets called. Also, since we are passing by value and this doesn't return anything, does it actually change the `a` variable?

This is also not a situation to use a method/receiver. This can be modeled as a pure function.